### PR TITLE
Additions to Seek test cases in preparation for adding SeekRPC

### PR DESCRIPTION
**Pull request by gdoubleyew**
Add test cases in preparation for merging SeekRPC
Add test cases that build and use the sample breast-cancer dataset

### CODE REVIEW

File: scripts/seek/runParallelJobs.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def jobThread(work_queue, tid=0, returnCodeQueue=None, outputQueue=None):
     except queue.Empty:
         pass
     return
-    
+
```
Comment (atadych) [inline scripts/seek/runParallelJobs.py:57]: There is syntax error on lines with:
`cmd := work_que` or  `job := work_queue.get(block=False):`

Comment (gdoubleyew) [reply scripts/seek/runParallelJobs.py:57]: So this is a new syntax added in Python 3.8. Can you update your conda python version?

Comment (atadych) [reply scripts/seek/runParallelJobs.py:57]: Ok, that helped

File: tests/pytestHelper.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+sleipnirBin = os.path.join(sleipnirDir, 'Debug')
+seekScriptsDir = os.path.join(sleipnirDir, 'scripts', 'seek')
+
+def createSampleDatabase():
+    dbDir = os.path.join(sampleBcDir, 'db')
+    dbFiles = glob.glob1(dbDir, '*.db')
+    if len(dbFiles) < 100:
+        os.makedirs(sampleBcDir, exist_ok=True)
+        inputBCFiles = os.path.join(testInputsDir, 'breast-cancer-sample')
+        os.system(f'cp -a {inputBCFiles}/* {sampleBcDir}/')
```
Comment (atadych) [inline tests/pytestHelper.py:19]: Tiny thing - I see some inconsistency with BC abbreviation, once it is `inputBCFiles`, the other time `sampleBcDir`

Comment (gdoubleyew) [reply tests/pytestHelper.py]: Ok, I improved the variable name and added a comment. This is where the "reference" breast cancer sample files (such as PCL files) are copied from the checked-in input directory to a temporary output directory in preparation for building the database.

**Review by atadych — APPROVED** [review]
I think it looks good now; all the tests went fine.

**Pull request merged**